doc/user_guide/tutorials_and_examples.rst

```diff
@@ -783,6 +783,14 @@ compilation instructions are in the ``README.md`` file, including how
 to switch from using the stand-alone extraction library to the NetCDF-based
 one (see :ref:`extraction_libraries` for details).
 
+Example 6: Read-only Verification
```

Comment:
I realise we have duplication between this text and the markdown README files. If you're feeling keen, we could experiment with removing this section and replacing it with an include of the appropriate README. Sphinx's `include` directive supports a 'parser' option which apparently allows this (https://docutils.sourceforge.io/docs/ref/rst/directives.html#include-options). However, now that I write all that, I think perhaps that would be a separate issue.

Reply:
I have it nearly all sorted out, then I realised that the rst has a link target from the dev guide, which would now need to be defined in the README.md :cry: ... I kind of ran out of time to see if this is possible, and opened #2847 with the patch I have done so far.

Reply:
Thanks Joerg :-)